noxfile.py

```diff
@@ -273,7 +273,8 @@ def showcase_mypy(
     """Perform typecheck analysis on the generated Showcase library."""
 
     # Install pytest and gapic-generator-python
-    session.install("mypy")
+    # Pin to 0.89.0 until fix is released for https://github.com/python/mypy/issues/10596
+    session.install("mypy==0.812")
```

Comment:
https://pypi.org/project/mypy/0.901/, just now released, is supposed to have a fix for that bug.

Reply:
Ah great! I'll retry my other PRs then. :D